user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -424,6 +424,126 @@ const [mockEvent2] = getMockHandlerInput(myContract);
 expect(mockEvent1).toEqual(mockEvent2); // Will always be true
 ```
 
+### Generate OpenAPI documentation from the contracts
+
+With the `getOpenApiDocument` feature, you can generate an OpenAPI documentation of your api from the contracts it provides to define its endpoints.
+
+```ts
+import { getOpenApiDocument } from '@swarmion/serverless-contracts';
+
+const openApiDocumentation = getOpenApiDocumentation({
+  title: 'Test API',
+  description: 'API description',
+  contracts: [getUserContract, postUserContract, deleteUserContract],
+});
+```
+
+The resulting value will be something like this OpenAPI documentation (depending on your contracts)
```

Comment:
Can you state that the result will be a JSON object?

I fear that the fact that you showcase `yml` might be misleading for the readers.